management-service/src/main/java/pl/pwr/zpi/reports/dto/request/CreateReportRequest.java

```diff
@@ -0,0 +1,34 @@
+package pl.pwr.zpi.reports.dto.request;
+
+import pl.pwr.zpi.reports.enums.Accuracy;
+
+import java.util.List;
+
+public record CreateReportRequest(
+        String clusterId,
+        Accuracy accuracy,
+        Long sinceMs,
+        Long toMs,
+        List<Long> slackReceiverIds,
+        List<Long> discordReceiverIds,
+        List<Long> mailReceiverIds,
+        List<ApplicationConfiguration> applicationConfigurations,
+        List<NodeConfiguration> nodeConfigurations
+) {
+
+    public record ApplicationConfiguration(
+            String applicationName,
+            String customPrompt,
+            boolean enabled,
```

Comment:
Do we need an enabled attrubute? As far as I understand, frontend will only send applications which are "enabled", the rest is assumed as "disabled". Reports service acts accordingly, so if a application or node is not provided, then it's logs will be skipped.